spring-cloud-deployer-local/src/main/java/org/springframework/cloud/deployer/spi/local/AbstractLocalDeployerSupport.java

```diff
@@ -73,6 +75,13 @@ final protected LocalDeployerProperties getLocalDeployerProperties() {
 		ArrayList<String> commands = new ArrayList<String>();
 		commands.add(properties.getJavaCmd());
 		Map<String, String> deploymentProperties = request.getDeploymentProperties();
+
+		// Adds Java System Properties (ie -Dmy.prop=val) before main class or -jar
+		if (deploymentProperties.containsKey("JAVA_OPTS")) {
+			String[] javaOpts = StringUtils.delimitedListToStringArray(deploymentProperties.get("JAVA_OPTS"), ",", " \n\r\t");
```

Comment:
This might be too aggressive in terms of deleting chars. Perhaps `tokenizeToStringArray` is a better option?